### DISCUSSION TITLE
Disable disclosure keyboard interaction

### DISCUSSION THREAD

**Discussion by abcmoritz** [tailwindlabs/headlessui#2029] [Help]
Hello,
is it possible to disable keyboard interactions for the disclosure component in Vue?
We don't need / want them because they don't work with an input field inside the toggle button.

I couldn't find a simple default property which deactivates them.

**Comment by startupmasters** [comment]
Did you figure this out?

**Reply by abcmoritz** [reply]
Poorly I don't even remember when I used this. 🫠
